ROADMAP.md

```diff
@@ -200,7 +200,7 @@ I will point out those changes inside the migration guide.
 - Touch Events
 - Scrolling
 - Optimized theme colors
-- Vuelidate and Vee Validate integration
```

Comment:
This vuelidate is a js module for validation: https://vuelidate.js.org/, So you need to revert it

Reply:
yes the change is done.